Assets/Plugins/iOS/Shopify/PaymentSession.swift

```diff
@@ -205,10 +209,24 @@ extension PaymentSession: PKPaymentAuthorizationViewControllerDelegate {
     }
 }
 
+@available(iOS 11.0, *)
+extension PaymentSession {
+    
+    func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationViewController, didAuthorizePayment payment: PKPayment, handler completion: @escaping (PKPaymentAuthorizationResult) -> Void) {}
+    
+    public func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationViewController, didSelect shippingMethod: PKShippingMethod, handler completion: @escaping (PKPaymentRequestShippingMethodUpdate) -> Swift.Void) {}
+    
+    public func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationViewController, didSelectShippingContact contact: PKContact, handler completion: @escaping (PKPaymentRequestShippingContactUpdate) -> Swift.Void) {}
+    
+    
+    public func paymentAuthorizationViewController(_ controller: PKPaymentAuthorizationViewController, didSelect paymentMethod: PKPaymentMethod, handler completion: @escaping (PKPaymentRequestPaymentMethodUpdate) -> Swift.Void) {}
```

Comment:
Right, I should of clarified this in the description.

In the new delegate calls, `PKPaymentRequestShippingContactUpdate` and `PKPaymentAuthorizationResult` classes expects an array of `PKError` for initialization. Since errors are marshalled in from managed code, we have not implemented the functionality for creating `PKError` from the Storefront APIs.